docs/static/settings/configuration-wildcard-pipeline-id.asciidoc

```diff
@@ -0,0 +1,19 @@
+[role="xpack"]
+[[wildcard-in-pipeline-id]]
+==== Wildcard support in pipeline ID
+++++
+<titleabbrev>Wildcard support in pipeline ID</titleabbrev>
+++++
+Pipeline ID must begin with a letter or underscore and contain only letters, underscores, dashes, and numbers.
+You can use `*` in `xpack.management.pipeline.id` to match any number of letter, underscore, dash, and number.
+
+[source,shell]
+-----
+xpack.management.pipeline.id: ["*logs", "*apache*", "tomcat_log"]
+-----
+
+In this example, `"*logs"` matches all IDs end in `logs`. `"*apache *"` matches any IDs with `apache` in the name.
```

Comment:
how to escape `*` in asciidoc?  `"*apache*"` bold apache instead of showing two `*` in text.

Reply:
It appears to be rendering correctly.  You must have figured this out.